app/views/api/ads/index.json.jbuilder

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+json.array! @ads do |ad|
+  json.id ad.id
+  json.kind ad.kind
+  json.category ad.category
+  json.address ad.address
+  json.zip ad.zip
+  json.city ad.city.name
+  json.phone ad.phone
+  json.description ad.description
```

Comment:
Not sure if this is easier to read:

```suggestion
  json.extract! ad, :id, :kind, :category, :address, :zip, :phone, :description
  json.city ad.city.name
```

or

```suggestion
  json.extract! ad, :id, :kind, :category, :address, :zip
  json.city ad.city.name
  json.extract! ad, :phone, :description
```

Reply:
Like it. Do not have much experience with jbuilder so all suggestions are welcome